docs/oscar_references.bib

```diff
@@ -6,3 +6,28 @@ @article{Tay87
   year=1987,
   pages={76--85}
 }
+
+@book{GP07,
+author = {G. Greuel and G. Pfister},
+title = {A Singular introduction to commutative algebra. With contributions by Olaf Bachmann, Christoph Lossen and Hans Sch"onemann. 2nd extended ed.},
+url = {http://link.springer.com/book/10.1007%2F978-3-540-73542-7},
+language = {English},
+pages = {xx   689},
+publisher = {Berlin: Springer},
+year = {2007},
+edition = {2},
+refereed = {0}
```

Comment:
What does this bibkey/value mean?  That this is not a peer reviewed publication?

Reply:
Do not know. Just took official Springer bibtex into mouse.